src/utils/utils.ts

```diff
@@ -73,19 +74,23 @@ export const loadNFT1155Contract = (config: ConfigEntry, address: string | undef
 }
 
 export const getNFTAddressFromInput = (
-  nftAddress: string,
+  nftAddress: string | undefined,
```

Comment:
functions will several args I suggest to pass an object with all the parameters as args
```ts
export const getNFTAddressFromInput = ({
 nftAddress,
 ddo,
 serviceType = 'nft-sales',
conditionType = 'transferNFT'
}: {
  nftAddress?: string,
 ddo: DDO,
 serviceType: ServiceType,
 conditionType: ConditionType
})
```

Reply:
I think in this case that would be a bit overkill because the methods uses just 4 parameters